systemtest/src/test/java/io/strimzi/systemtest/mirrormaker/MirrorMaker2ST.java

```diff
@@ -893,6 +900,170 @@ void testConfigureDeploymentStrategy() {
+        JobUtils.deleteJobWithWait(NAMESPACE, targetConsumerName);
+
+        LOGGER.info("Send 50 messages to Source cluster");
+        internalClientSourceJob = internalClientSourceJob.toBuilder().withMessageCount(50).withMessage("Producer C").build();
+        internalClientSourceJob.createAndWaitForReadiness(internalClientSourceJob.producerStrimzi().build());
+        ClientUtils.waitForClientSuccess(sourceProducerName, NAMESPACE, 50);
+        JobUtils.deleteJobWithWait(NAMESPACE, sourceProducerName);
+
+        waitForMM2OffsetIntervalMirroring(syncGroupOffsetsIntervalSeconds);
+
+        LOGGER.info("Receive 10 msgs from source cluster");
+        internalClientSourceJob = internalClientSourceJob.toBuilder().withMessageCount(10).withAdditionalConfig("max.poll.records=10").build();
+        internalClientSourceJob.createAndWaitForReadiness(internalClientSourceJob.consumerStrimzi().build());
+        ClientUtils.waitForClientSuccess(sourceConsumerName, NAMESPACE, 10);
+        JobUtils.deleteJobWithWait(NAMESPACE, sourceConsumerName);
+
+        waitForMM2OffsetIntervalMirroring(syncGroupOffsetsIntervalSeconds);
+
+        LOGGER.info("Receive 40 msgs from mirrored topic on Target cluster");
+        KafkaBasicExampleClients internalClientTargetJob = initialInternalClientTargetJob.toBuilder().withMessageCount(40).build();
+        internalClientTargetJob.createAndWaitForReadiness(internalClientTargetJob.consumerStrimzi().build());
+        ClientUtils.waitForClientSuccess(targetConsumerName, NAMESPACE, 40);
+        JobUtils.deleteJobWithWait(NAMESPACE, targetConsumerName);
+
+        LOGGER.info("There should be no more messages to read. Try to consume at least 1 message. " +
+                "This client job should fail on timeout.");
+        initialInternalClientTargetJob.createAndWaitForReadiness(initialInternalClientTargetJob.consumerStrimzi().build());
+        assertThrows(WaitException.class, () -> ClientUtils.waitForClientSuccess(targetConsumerName, NAMESPACE, 1));
+
+        LOGGER.info("As it's Active-Active MM2 mode, there should be no more messages to read from Source cluster" +
+                " topic. This client job should fail on timeout.");
+        initialInternalClientSourceJob.createAndWaitForReadiness(initialInternalClientSourceJob.consumerStrimzi().build());
+        assertThrows(WaitException.class, () -> ClientUtils.waitForClientSuccess(sourceConsumerName, NAMESPACE, 1));
+    }
+
+    void waitForMM2OffsetIntervalMirroring(String timeout) {
+        // wait sync.group.offsets.interval.seconds to sync in target cluster
+        LOGGER.info("Wait {} secs -> sync.group.offsets.interval.seconds, check msgs in target cluster", timeout);
+        try {
+            Thread.sleep(Duration.ofSeconds(Long.parseLong(timeout)).toMillis());
+        } catch (InterruptedException e) {
+            LOGGER.error("Error while waiting on syncGroupOffsetsIntervalSeconds");
+        }
+    }
+
```

Comment:
Why is this needed? Isn't better option to extend the timeout for clients? Even if it's just for one sec?

Reply:
Imo, this gives test more readability about what's going on. Sure I can prolong client's timeout.

Reply:
what about comment add just comment instead of this weird sleep? Or wait for some event